### ISSUE TITLE
migrate docker builds to github

### ISSUE THREAD

**Issue by kevinkreiser** [valhalla/valhalla#4029]
docker is nuking all free accounts, which is cool with me. one less place to care about. good news is github supports docker builds directly already. so yeah lets just migrate to githubs docker container registry: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry

**Comment by nilsnolde** [comment]
It’s not quite as bad yet. What they’re nuking is free team accounts. If they did what you think they did, they might as well shut down all their stuff and join Travis CI (anyone remember haha I had a really hard time remembering their name) in the row of unimportant tech companies. But they’re clearly inching towards that.

**Comment by kevinkreiser** [comment]
that was what i thought at first as well but when i read their docs its was wholly unclear what a "team account" was and how the change impacted our use case: open source project in an organization. since i wanted to move to github anyway i figured who cares.

hilariously, i tried to find again the announcement that they had and instead the top search result was: https://www.docker.com/blog/we-apologize-we-did-a-terrible-job-announcing-the-end-of-docker-free-teams/

basically:

> we are stupid and confused everyone

which again, im totally cool with, i like github and would still advocate for migrating there (unless there is good reason not to)

**Comment by kevinkreiser** [comment]
stranger still that clarification blog says:

>  any affected organizations are labeled “Docker Free Team” in the “Subscription” column.

and indeed our orgs are marked in the way that they deem "affected":

![image](https://user-images.githubusercontent.com/697548/226024625-9d8b657f-c5e8-41c8-b64f-35143b834ef2.png)

so yeah i stand by what i said, forget 'em i like github better anyway :smile:

**Comment by nilsnolde** [comment]
Hahaha wtf, that’s ridiculous.. but true, „teams“ are of course orgs like us.. Goodbye docker then.. hope MS won’t crap up GitHub like that too. Gitlab is getting closer to what Docker is doing, soon there won’t be any more choice anyways.

**Comment by kevinkreiser** [comment]
![image](https://user-images.githubusercontent.com/697548/227664365-1bafa64f-7d9f-4ddb-acde-7126b6ce7377.png)

too late, bye bye ;)

**Comment by nilsnolde** [comment]
Hahahaha but respect, that’s pretty embarrassing all in all, and they owned it.
